### DISCUSSION TITLE
Teams notification (webhook)

### DISCUSSION THREAD

**Discussion by vb4life** [upptime/upptime#285] [Ideas] [answered]
Any plans for supporting Teams "Incoming webhook" as a means of notifications?
https://docs.microsoft.com/en-us/outlook/actionable-messages/send-via-connectors
https://docs.microsoft.com/en-us/outlook/actionable-messages/message-card-reference

Just a basic functionality would be great!

**Comment by AnandChowdhary** [comment, marked as answer]
We're currently using https://github.com/notifme/notifme-sdk as the SDK for notifications. AFAIK, they don't have support for Teams yet. Do you think you can open an issue there and we'll see if they're interested in adding it?

I agree that Teams is a popular comms tool these days, so if they're not interested in picking this up, perhaps I can make a PR there or move away to a different notifications tool.

**Reply by vb4life** [reply]
There is an old issue there since 2017: https://github.com/notifme/notifme-sdk/issues/2
In 2019 they requested help. So it doesn't seem plausible that it will be made.

The project seems quite dead since about 2 years back.

**Comment by dougwaldron** [comment]
While this would be nice to have, a good alternative is to add the GitHub integration in Teams and subscribe to new issues in your Upptime repo:
[GitHub + Microsoft Teams](https://teams.github.com/)

**Comment by sercanuste** [comment]
Hi @vb4life. Teams integration with incoming webhook is released at [v1.28.0](https://github.com/upptime/uptime-monitor/releases/tag/v1.28.0). You can find instructions at [documentation](https://upptime.js.org/docs/notifications#microsoft-teams).
